src/Model/IPaymentMethod.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace ThePay\ApiClient\Model;
+
+use ThePay\ApiClient\ValueObject\Url;
+
+interface IPaymentMethod
+{
+    /**
+     * @return string
+     */
+    public function getCode();
+
+    /**
+     * @return string
+     */
+    public function getTitle();
+
+    /**
+     * @return Url
+     */
+    public function getImageUrl();
+
+    /**
+     * @return array<string>
+     */
+    public function getTags();
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function toArray();
```

Comment:
I not found any usage of `toArray` method from this interface, if is not important for interface remove it.

Reply:
does not to be in interface, so i will remove it